functions.yml

```diff
@@ -187,26 +200,23 @@ etl:
           doc: Location of the chain file on each node in the cluster
```

Comment:
For above: replace the doc for `The current contigName` -> `The current contig name`

Reply:
Done